runtime/xamarin/coreclr-bridge.h

```diff
@@ -6,6 +6,9 @@
 #ifndef __CORECLR_BRIDGE__
 #define __CORECLR_BRIDGE__
 
+//#define LOG_CORECLR(...)
+#define LOG_CORECLR(...) fprintf (__VA_ARGS__)
```

Comment:
What's the diff between LOG_CORECLR and LOG_MONOVM ?

Reply:
There's no difference really, except that one is used in corelcr-bridge.m and the other in monovm-bridge.m.